### PR TITLE
PR from issue 2

### PR DESCRIPTION
**Pull request by nmccready**
#2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 'use strict';
 
 module.exports = require('./dist/promiseFtp');
-module.exports.FtpConnectionError = require('promise-ftp-common').FtpConnectionError;
-module.exports.FtpReconnectError = require('promise-ftp-common').FtpReconnectError;
-module.exports.STATUSES = require('promise-ftp-common').STATUSES;
+module.exports.FtpConnectionError = require('@motiz88/promise-ftp-common').FtpConnectionError;
```
Comment (nmccready) [inline index.js]: @motiz88 back this out

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "promise-ftp",
+  "name": "@motiz88/promise-ftp",
```
Comment (nmccready) [inline package.json]: resolve conflicts

**Comment by fractalawareness** [comment]
Hello! Is any progress expected on that?

**Comment by zacronos** [comment]
Hi @fractalawareness -- are you encountering the same issue (#2)?  One difficulty on my side is I don't have a good test showing the problem.

**Comment by RedHatter** [comment]
I ran into this problem with a race condition.

    async function () {
     let ftp = new PromiseFtp()
     await ftp.connect({host, user, password})
     ftp.get('file').then(console.log)
     await ftp.end()
    }

If `end` gets called before `get` is done I get `{ FtpConnectionError: can't perform 'get' command when connection status is: disconnecting
` with @motiz88 fork while the original fails silently.

**Comment by zacronos** [comment]
If someone would like to create another PR without the merge conflicts, I'd be happy to review and merge it.

**Comment by zacronos** [comment]
Has been fixed by another PR.

**Pull request closed without merging**